src/wasm-ir-builder.h

```diff
@@ -395,16 +398,17 @@ class IRBuilder : public UnifiedExpressionVisitor<IRBuilder, Result<>> {
     static ScopeCtx makeLoop(Loop* loop, Type inputType) {
       return ScopeCtx(LoopScope{loop}, inputType);
     }
-    static ScopeCtx makeTry(Try* tryy, Name originalLabel, Type inputType) {
-      return ScopeCtx(TryScope{tryy, originalLabel}, inputType);
+    static ScopeCtx
+    makeTry(Try* tryy, Name originalLabel, Type inputType, Index index) {
+      return ScopeCtx(TryScope{tryy, originalLabel, index}, inputType);
```

Comment:
It would never make sense to create a `Try` scope with any index other than 0.
```suggestion
    makeTry(Try* tryy, Name originalLabel, Type inputType) {
      return ScopeCtx(TryScope{tryy, originalLabel, 0}, inputType);
```